aws/table_aws_cloudwatch_log_group.go

```diff
@@ -195,22 +176,28 @@ func getCloudwatchLogGroup(ctx context.Context, d *plugin.QueryData, _ *plugin.H
 func getLogGroupTagging(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
 	logGroup := h.Item.(types.LogGroup)
 
+	// https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/iam-access-control-overview-cwl.html
+	// DescribeLogGroups API returns the logGroup arn format arn:aws:logs:region:account-id:log-group:log_group_name:*
+	// ListTagsForResource API support the logGroup arn format arn:aws:logs:region:account-id:log-group:log_group_name
+	logGroupArn := strings.TrimSuffix(*logGroup.Arn, ":*")
```

Comment:
@bigdatasourav Do you know why AWS appends `:*` at the end of the ARN for the DescribeLogGroups API? What does the `:*` represent?

Reply:
@cbruno10 It represents all the log streams present in that log group.

Reply:
Log stream arn format -- arn:aws:logs:region:account-id:log-group:log_group_name:log-stream:log-stream-name